docs/podman-create.1.md

```diff
@@ -464,6 +477,13 @@ interactive shell. The default is false.
 Note: The **-t** option is incompatible with a redirection of the podman client
 standard input.
 
+**--uidmap**=map
+   UID map for the user namespace.  Using this flag will run the container with user namespace enabled, unless the --userns=host flag is turned on.
```

Comment:
Maybe it's me, but I'm confused by the "unless the --userns=host flag is turned on." bit here and in --gidmap,  --subgidname and --subuidname sections.  Especially when you look at --userns and it says "unless you turn on --uidmap or --gidmap."  A number of examples over all of this would be very useful.

Reply:
I think we should conflict the options.

--userns=container:FOOBAR is the only option that makes sense for userns,  Unless we add an option to libpod.conf to specify a default usermap.  (Which we don't have yet).

If you don't specify --uidmap or --gidmap then you get no userns enabled.

Reply:
@rhatdan I don't think specifying default userns values should be a libpod thing - we should let the people who call us do that, like we do for Seccomp and such. If CRI-O or Podman want to have their own defaults that would be fine.

Reply:
Thats fine. I just want a comman config file that podman and buildah can share.  But for now lets just ignore this and force users to specify the command line.